community/io/src/main/java/org/neo4j/io/pagecache/impl/muninn/MuninnPageCursor.java

```diff
@@ -37,6 +39,9 @@
     private static final boolean tracePinnedCachePageId =
             flag( MuninnPageCursor.class, "tracePinnedCachePageId", false );
 
+    private static final boolean usePreciseCursorErrorStackTraces =
```

Comment:
do you think we will ever need non precise stack traces? what will be use case of that?